backend/e4h-services/ingestion-service/app/utils/amc_scheduler_service_client.py

```
@@ -58,6 +58,46 @@ def create_amc_configuration(self, request_info: RequestInfo, configuration_payl
+        logger.trace(f"Creating AMC configurations in bulk: count={len(configuration_payloads)}")
+        url = f"{self.amc_scheduler_service_url}/asset-amc/v1/configuration/_create"
+        headers = {
+            "Content-Type": "application/json"
+        }
+        payload = {
+            "RequestInfo": request_info.model_dump(by_alias=True, exclude_none=True),
+            "AmcConfigurations": configuration_payloads,
+            "apiOperation": "CREATE"
+        }
+        try:
+            response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)